modules/ROOT/pages/app-dashboards.adoc

```diff
@@ -95,87 +82,553 @@ The following charts are available only if you are using Unified Agent and have
+
+Total Failed Inbound Requests by Endpoint::
+The sum of failed inbound requests, grouped by endpoint.
+
+Failed Outbound Requests: Response Time by HTTP endpoint::
+The average, maximum, and minimum response times of failed outbound requests, grouped by endpoint. Cells will vary in color according to the following thresholds:
+* Average: Green if the response time is less than 300ms; yellow if the response time is greater than 300ms and less than 600ms; red if the response time is greater than or equal to 600ms.
+* Maximum: Green if the response time is less than 300ms; yellow if the response time is greater than 300ms and less than 500ms; red if the response time is greater than or equal to 500ms.
+* Minimum: Green if the response time is less than 300ms; yellow if the response time is greater than 300ms and less than 600ms; red if the response time is greater than or equal to 600ms.
+
+Total Failed Outbound Requests by HTTP Endpoint::
+The sum of failed outbound requests, grouped by endpoint.
+
+[[jvm-charts]]
+== JVM
+
+The following charts appear on the *JVM* dashboard:
+
+Garbage Collection Count::
+The number of collections that have occurred, grouped by worker in automatic intervals.
+
+Garbage Collection Time::
+The approximate accumulated collection elapsed time in milliseconds, grouped by type of response in automatic intervals.
+
+Par New Collection Count::
+The number of collections for Parnew GC, grouped by type of response in automatic intervals.
+
+Par New Collection Time::
+The approximate accumulated Parnew collection elapsed time in milliseconds, grouped by type of response in automatic intervals.
+
+Classes Loaded::
+The number of currently loaded classes, grouped by type of response in automatic intervals.
+
+Classes Loaded Total::
+The number of classes loaded since JVM start, grouped by type of response in automatic intervals.
+
+Classes Unloaded::
+The number of classes unloaded since JVM start, grouped by type of response in automatic intervals.
+
+Heap Committed::
+The amount of heap memory committed by the JVM, grouped by type of response in automatic intervals.
+
+Heap Used::
+The amount of heap memory utilized by the JVM, grouped by type of response in automatic intervals.
+
+Thread Count::
+The number of simultaneous requests.
+
+JVM Uptime::
+The time since JVM started, grouped by type of response in automatic intervals.
+
+Par Eden graphs (Used, Max, Init, and Committed)::
+The usage metrics for “Eden Space” JVM Memory, grouped by type of response in automatic intervals.
+
+Par Survivor graphs (Used, Max, Init, and Committed)::
+The usage metrics for the Par Eden Space pool.
+
+Metaspace graphs (Used, Max, Init, Committed)::
+The usage metrics for Java memory space.
+
+Code Cache graphs (Used, Max, Init, Committed)::
+The usage metrics for the Code Cache pool.
+
+Compressed Class Space graphs (Used, Max, Init, Committed)::
+The usage metrics for values of the metadata for compressed class pointers.
```

Comment:
The memory metrics for JVM compressed class grouped by worker in automatic intervals.

Reply:
```suggestion
The memory metrics for JVM compressed class grouped by worker in automatic intervals.
```